CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to Bmap-rs
+
+:+1: Thank you for contributing! :+1:
+These are mostly guidelines, not rules. Use your best judgment and follow
+these guidelines when contributing to the project.
+
+## How to Contribute
+
+- Bugs: Tracked as ([GitHub issues](https://github.com/collabora/bmap-rs/issues)).
+- Enhancements: RFE suggestions are tracked as
+([GitHub issues](https://github.com/collabora/bmap-rs/issues)).
+- Code: Managed on [GitHub](https://github.com/collabora/bmap-rs) through
+  Pull Requests ([PR](https://github.com/collabora/bmap-rs/pulls)).
+
+
+## Code structure
```

Comment:
drop this section; it doesn't add any benefit yet. We can always add it later :-)